Stripe/STPBankSelectionTableViewCell.m

```diff
@@ -95,6 +96,10 @@ - (void)configureWithBank:(STPFPXBankBrand)bankBrand theme:(STPTheme *)theme sel
     // Title label
     self.titleLabel.font = theme.font;
     self.titleLabel.text = STPStringFromFPXBankBrand(self.bank);
+    if (offline) {
+        NSString *format = STPLocalizedString(@"%@ - Offline", @"Bank name when bank is offline for maintenance.");
+        self.titleLabel.text = [NSString stringWithFormat:format, STPStringFromFPXBankBrand(self.bank)];
```

Comment:
Do you think it's worthwhile to just always use `[NSString localizedStringWithFormat:]`?

Reply:
STPLocalizedString does some bundle searching logic to allow for users to override our localizations. We'd want a similar macro for `localizedStringWithFormat:` if we wanted to start using that.